cloudinary-image-management-and-manipulation-in-the-cloud-cdn/js/src/components/sync.js

```diff
@@ -118,6 +118,16 @@ const Sync = {
 
 export default Sync;
 
+// Add it a trigger watch to stop deactivation.
+let triggers = document.getElementsByClassName( 'cld-deactivate' );
+[ ...triggers ].map( ( trigger ) => {
```

Comment:
Since you're not changing the contents of the array, better to use `forEach` here.

Reply:
@dugajean - please make a suggestion on this. My JS knowledge is lacking and this is the best way I know how to do it :)